server/manager/src/main/java/org/apache/accumulo/manager/state/SetGoalState.java

```diff
@@ -36,7 +36,15 @@ public class SetGoalState {
    * Utility program that will change the goal state for the manager from the command line.
    */
   public static void main(String[] args) throws Exception {
-    if (args.length != 1 || ManagerGoalState.valueOf(args[0]) == null) {
+    if (args.length != 1) {
+      System.err.println(
+          "Usage: accumulo " + SetGoalState.class.getName() + " [NORMAL|SAFE_MODE|CLEAN_STOP]");
+      System.exit(-1);
+    }
+
+    try {
+      ManagerGoalState.valueOf(args[0]);
```

Comment:
updated errorprone flagged this is an impossible null check because Enum.valueOf does not return null, it throws an IllegalArgumentException.